workflows-scripts/circulation/check-in-check-out/scripts/checkin-checkout-db-restore.sql

```diff
@@ -0,0 +1,20 @@
+/*
+To use this script, enter the command:
+$ psql -f checkin-checkout-db-restore.sql -a --echo-all
+Also supply the values for -h, -d, -U if applicable
+
+This script will remove all the loans and requests, and their related data from the tables. It is expected that these loans and requests will be repopulated by another script.

```

Comment:
When you say "another script" do you mean populate.sh?

Reply:
It could be populate.sh or something else, which is independent of this script.